streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/AssignmentInfo.java

```diff
@@ -228,6 +256,11 @@ public static AssignmentInfo decode(final ByteBuffer data) {
                     assignmentInfo = new AssignmentInfo(usedVersion, latestSupportedVersion);
                     decodeVersionThreeData(assignmentInfo, in);
                     break;
+                case 4:
+                    final int latestSupportedVer = in.readInt();
```

Comment:
nit: latestSupportedVersion

Reply:
I did that first - resulting in duplicate local variable.

Reply:
I see --- we can define latestSupportedVersion before the switch not as a `final` int then?